main.go

```diff
@@ -99,19 +99,23 @@ func main() {
 		}
 
 		if output.Path != nil && output.Name != "" {
-			fmt.Printf("Added %v \n", output.Name)
+			_, _ = fmt.Fprintln(os.Stderr, fmt.Sprintf("Added %v", output.Name))
 		}
 	}
 
 	if err := <-errCh; err != nil {
 		_, _ = fmt.Fprintln(os.Stderr, err)
-		os.Exit(-1)
+		exit(start, -1)
```

Comment:
Should be non-negative

Reply:
Hmm interesting, we have `os.Exit(-1)` all over this file. Should we change this everywhere?